manifests/attribute_filter.pp

```diff
@@ -7,10 +7,11 @@
 
   # Create the attribute-filter.xml configuration file.
   concat { 'attribute-filter.xml':
-    path  => "${shibboleth_idp::shib_install_base}/conf/attribute-filter.xml",
-    owner => $shibboleth_idp::shib_user,
-    group => $shibboleth_idp::shib_group,
-    mode  => '0600',
+    path   => "${shibboleth_idp::shib_install_base}/conf/attribute-filter.xml",
+    owner  => $shibboleth_idp::shib_user,
+    group  => $shibboleth_idp::shib_group,
+    mode   => '0600',
+    notify => Class['shibboleth_idp::service'],
```

Comment:
Yes. I just tested again. If a fragment changes, the class is correctly notified.

My biggest concern is triggering too often since Jetty takes a long time to restart. I've testing multiple provisioning runs without any changes and so far it looks good, but I'll want to watch this.